README.md

```diff
@@ -84,14 +84,14 @@ var url = client.buildAuthorizationEndpoint(redirect_uri, scopes, state)
 
 Name           | Type     | Description
 ---------------|----------|------------
-`redirect_uri` | string   | The URI the user is redirected to in step 3. This value must be provided to every step of the authorization process including creating your app, building your authorization endpoint and exchanging your authorization code for an access token
-`scope`        | array    | An array of permissions your token needs to access. You can read more at https://developer.vimeo.com/api/authentication#supported-scopes
-`state`        | string   | A value unique to this authorization request. You should generate it randomly, and validate it in step 3.
+`redirect_uri` | string   | The URI the user is redirected to in Step 3. This value must be provided to every step of the authorization process, including creating your app, building your authorization endpoint, and exchanging your authorization code for an access token.
+`scope`        | array    | An array of permissions your token needs to access. You can read more at https://developer.vimeo.com/api/authentication#supported-scopes.
+`state`        | string   | A value unique to this authorization request. You should generate it randomly and validate it in Step 3.
 
-2. Your user will need to access the authorization endpoint (either by cliking the link or through a redirect). On the authorization endpoint the user will have the option to deny your app any scopes you have requested. If they deny your app, they will be redirected back to your `redirect_url` with an `error` parameter.
+2. Your user needs to access the authorization endpoint (either by clicking the link or through a redirect). On the authorization endpoint, the user will have the option to deny your app any scopes you have requested. If they deny your app, they are redirected back to your `redirect_url` with an `error` parameter.
```

Comment:
Ah, I might have missed this one. Can we say _the user has the option_ instead of _the user will have the option_ here?